packages/nodes-base/nodes/Jwt/Jwt.node.json

```diff
@@ -0,0 +1,19 @@
+{
+	"node": "n8n-nodes-base.jwt",
+	"nodeVersion": "1.0",
+	"codexVersion": "1.0",
+	"categories": ["Development"],
+	"alias": ["Token", "Key", "JSON", "Payload", "Sign", "Verify", "Decode"],
+	"resources": {
+		"credentialDocumentation": [
+			{
+				"url": "https://docs.n8n.io/credentials/jwt"
+			}
+		],
+		"primaryDocumentation": [
+			{
+				"url": "https://docs.n8n.io/integrations/builtin/app-nodes/n8n-nodes-base.jwt/"
```

Comment:
This should probably be `https://docs.n8n.io/integrations/builtin/core-nodes/n8n-nodes-base.jwt/` as it isn't integrating with a service